concordium-android-sdk/pom.xml

```
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.concordium.sdk</groupId>
         <artifactId>concordium-sdk-base</artifactId>
-        <version>7.2.0</version>
+        <version>7.3.0-SNAPSHOT</version>
```

Surely these will be breaking changes, right? Maybe you should 
```suggestion
        <version>8.0.0-SNAPSHOT</version>
```

---

No breaking changes so far.

---

Actually, there are breaking changes in `BakerPoolStatus` getters, so I'll change the version to 8.0.0-SNAPSHOT